src/fides/api/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -264,15 +270,48 @@ def set_consent_preferences(
                 raise HTTPException(
                     status_code=HTTP_400_BAD_REQUEST, detail=Pii(str(exc))
                 )
+    identity = Identity()
+    setattr(
+        identity,
+        provided_identity.field_name.value,  # type:ignore[attr-defined]
+        provided_identity.encrypted_value["value"],  # type:ignore[index]
+    )
 
-    return _prepare_consent_preferences(db, provided_identity)
+    consent_preferences: ConsentPreferences = _prepare_consent_preferences(
+        db, provided_identity
+    )
+
+    privacy_request_results: BulkPostPrivacyRequests = create_privacy_request_func(
+        db=db,
+        data=[
+            PrivacyRequestCreate(
+                identity=identity,
+                policy_key=DEFAULT_CONSENT_POLICY,
+                consent_preferences=[
+                    consent.dict() for consent in consent_preferences.consent if consent
+                ],
+            )
+        ],
+        authenticated=True,
```

Comment:
We've authenticated with the existing consent request workflow, so now we can say this is authenticated when we create the privacy request to not send another email.